.github/workflows/dockerbuild.yml

```
@@ -35,13 +31,19 @@ jobs:
       - name: Available platforms
         run: echo ${{ steps.qemu.outputs.platforms }}
 
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: unifitoolbox/protect-archiver:latest
```

Let's make the tags a bit more specific:

```suggestion
          images: unifitoolbox/protect-archiver:latest
          tags: |
            type=edge
            type=semver,pattern={{version}}
```

If I understand the docs correctly, this should result in the label `edge` being added to images built from the latest push to the `master` branch, and for the labels `x.y.z` and `latest` to be added to images built for every new Semver Git tag.